src/company/company.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Company } from '../entity/company.entity';
+
+
+@Injectable()
+export class CompanyService {
+
+    constructor(
+        @InjectRepository(Company)
+        private companyRepository: Repository<Company>,
+
+        
+    ){}
+    
+    async createCompany(createcompanyDto: Company): Promise<Company> {
```

Comment:
We'll need to implement some protections here later so that only certain users can create a company

Reply:
Sure that is what I am gonna do this week, I guess